src/olympia/addons/tests/test_serializers.py

```diff
@@ -110,6 +110,7 @@ def test_basic(self):
             average_rating=4.21,
             bayesian_rating=4.22,
             category=cat1,
+            contributions=u'https://paypal.me/foobar/',
```

Comment:
stupid me, I wonder why we called the model field `contributions` and not `contributions_url` heh

Reply:
We (annoyingly) already have a `Addon.contribution_url` property, which is used for the current paypal implementation.  At the time it seemed easier to call it just `contributions`

Reply:
right, I forgot that